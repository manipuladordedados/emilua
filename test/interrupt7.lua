-- Read invalid interruption_caught on not-yet-had-a-chance-to-run-fiber
local println = require('println')

fib = spawn(function() end)
println(tostring(fib.interruption_caught))
