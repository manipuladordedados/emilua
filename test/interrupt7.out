Main fiber from VM 0x0 panicked: 'Invalid argument'
stack traceback:
	input.lua:5: in main chunk
